core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrepImp.java

```diff
@@ -0,0 +1,132 @@
+package ca.jrvs.apps.grep;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.log4j.BasicConfigurator;
+import java.io.*;
```

Comment:
It is better to have individual imports, so we can know what packages are we using.